metrics/test_driver/util/file_utils.dart

```diff
@@ -92,6 +102,8 @@ class FileUtils {
     }
   }
 
+  /// Method needed to save the process outputs from the [stderr] and [stdout]
```

Comment:
Start with an action, per the effective dart style guide:
https://dart.dev/guides/language/effective-dart/documentation

This would be:

```diff
-Method needed to save the process outputs ...
+Saves process output ...
```

Also please update the process doc https://github.com/software-platform/metrics/blob/master/docs/02_process.md to include this information about comment styles.